lib/kadai-core-test/src/test/java/acceptance/KadaiEngineInitalizationTest.java

```diff
@@ -55,6 +67,46 @@ public OptionalInt calculatePriority(TaskSummary taskSummary) {
     }
   }
 
+  private static class TestKadaiEngineImpl extends KadaiEngineImpl {
```

Comment:
Why do you need this class and can't just call `KadaiEngineImpl.createKadaiEngine`?

Reply:
You are right, I did not know we had that method